src/internal/auth/interceptor.go

```diff
@@ -266,7 +274,7 @@ func (i *Interceptor) InterceptStream(srv interface{}, stream grpc.ServerStream,
 		return fmt.Errorf("no auth function for %q, this is a bug", info.FullMethod)
 	}
 
-	if err := a(pachClient, info.FullMethod); err != nil {
+	if _, err := a(pachClient, info.FullMethod); err != nil {
```

Comment:
We should also be able to set the context for the incoming stream, so this works for streaming and unary calls.